test/test_images/autoscale/autoscale.go

```diff
@@ -151,6 +181,13 @@ func handler(w http.ResponseWriter, r *http.Request) {
 			fmt.Fprint(w, bloat(mb))
 		}()
 	}
+	if !hasMs && !hasMax && !hasMb {
```

Comment:
I think it's the querystring parameters to control the behavior that makes this image to useful.  Can we make the two flags you have querystring parameters instead?

Reply:
Perhaps if `sleep-stdev=x` is passed then `sleep` is used as the mean and the standard deviation is applied?

Reply:
Initially I was thinking to make the two values constant to get statistical significance, but as when we use this image in a test, the query string we use are always the same, I agree it makes sense to use query string instead.
Done with the change.